Formula/bzip2.rb

```diff
@@ -1,10 +1,8 @@
 class Bzip2 < Formula
   desc "Freely available high-quality data compressor"
   homepage "https://en.wikipedia.org/wiki/Bzip2"
-  url "https://ftp.osuosl.org/pub/clfs/conglomeration/bzip2/bzip2-1.0.6.tar.gz"
-  mirror "https://fossies.org/linux/misc/bzip2-1.0.6.tar.gz"
-  sha256 "a2848f34fcd5d6cf47def00461fcb528a0484d8edef8208d6d2e2909dc61d9cd"
-  revision 1
+  url "https://sourceware.org/pub/bzip2/bzip2-1.0.7.tar.gz"
```

Comment:
Both [Gentoo](https://github.com/gentoo/gentoo/blob/5d061ef62b8495c1bc984aa58c3c35775bcae031/app-arch/bzip2/bzip2-1.0.7.ebuild#L12-L13) and [MacPorts](https://github.com/macports/macports-ports/blob/429816eda9ae17aa0cd913618e0297caeb151a47/archivers/bzip2/Portfile#L23-L24) are using this `sourceware.org` URL.